### PR TITLE
feat: added body scrollview ref

### PR DESCRIPTION
**Pull request by pi0trpietruszewski**
Added body scrollview reference

### CODE REVIEW

**Review by uysalut — APPROVED** [review]

File: src/index.d.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { ReactElement, Component } from 'react';
-import { ImageSourcePropType, NativeScrollEvent, NativeSyntheticEvent, TextStyle, ViewStyle, ImageResizeMode } from 'react-native';
+import { ImageSourcePropType,HeaderTypeProp, ScrollView, NativeScrollEvent, NativeSyntheticEvent, TextStyle, ViewStyle, ImageResizeMode } from 'react-native';
```
Comment (marcinkornek) [inline src/index.d.ts:2]: pls add space after comma `ImageSourcePropType,HeaderTypeProp` to `ImageSourcePropType, HeaderTypeProp`

File: README.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ Below are examples of those components and description of the props they are acc
...
 | `renderBody`                  | `func`                                                |    Yes   | `title => <RenderContent title={title} />`                                                                                                                                                     | Function that renders body of the header (can be empty)  |
 | `snapToEdge`                  | `bool`                                                |    Yes   | `true`                                                                                                                                                                                         | Boolean to fire the function for snap To Edge            |
+| `scrollRef`                   | `func or object`                                      |    Yes   | `null`                                                                                                                                                                                         | ScrollView body ref.           |
```
Comment (marcinkornek) [inline README.md:78]: pls add more information what it can be used etc :)

File: src/index.d.ts (hunk 1 above)
Comment (pi0trpietruszewski) [reply src/index.d.ts]: done

File: README.md (hunk 2 above)
Comment (pi0trpietruszewski) [reply README.md]: done

**Review by marcinkornek — APPROVED** [review]

**Review by pwysowski — APPROVED** [review]

**Pull request merged**